### ISSUE TITLE
GraphQL does not support dotAssets in image/file fields

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#21445]
In dotCMS, when you create a content type with a image/file field, we allow a user to upload and select dotAssets as well as FileAssets.

The issue is that our graphql implementation is only expecting to deliver FileAssets. If a user has a dotAsset selected in a file/image field, we throw an error in graphql.

We need to provide a common interface for dotAssets/fileAssets that we can return for graphql queries. 

You can see in code where we expect only fileAssets:

https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotcms/graphql/datafetcher/FileFieldDataFetcher.java#L35

Current workarounds to this issue are to only use FileAssets in File/Image field or to change those fields to binary field types.

**Comment by victoralfaro-dotcms** [comment]
Tested with this request:
```query ContentAPI {
	FileAssetCollection {
    title
    tags
		contentType
		baseType
  }
}```

Got this response:
```{
	"errors": [
		{
			"message": "Validation error of type FieldUndefined: Field 'tags' in type 'FileAsset' is undefined @ 'FileAssetCollection/tags'",
			"locations": [
				{
					"line": 4,
					"column": 5
				}
			],
			"extensions": {
				"classification": "ValidationError"
			}
		}
	],
	"data": null
}```

**Comment by bryanboza** [comment]
Fixed, tested on release-22.02 // Docker // Insomnia
